pandas/tests/plotting/test_datetimelike.py

```diff
@@ -44,6 +44,10 @@ def teardown_method(self, method):
         tm.close()
 
     @pytest.mark.slow
+    @pytest.mark.filterwarnings(
+        "ignore:Converting to PeriodArray/Index representation "
```

Comment:
can you add the issue number that this warning comes from.

I would prefer that you separate these cases so its more explict. (as you do below)

Reply:
Where is "below"?

Reply:
you indicated another way of doing this in your comments,

Reply:
Ah okay, I hadn't realized you wanted that in. Will include it here, too.